[2020-11-23 13:00:28,037] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 13:03:15,876] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 14:14:57,041] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 14:31:37,112] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 14:34:48,756] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 14:38:10,213] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 14:38:41,651] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 14:40:28,363] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 14:57:08,337] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 15:18:02,894] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 15:58:47,025] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-23 16:00:40,571] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-24 16:16:16,784] SEVERE {b7a.log.crash} - java.lang.NullPointerException 
org.ballerinalang.jvm.util.exceptions.BallerinaException: java.lang.NullPointerException
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.runOnSchedule(TesterinaFunction.java:125)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.invoke(TesterinaFunction.java:56)
	at org.ballerinalang.test.runtime.BTestRunner.invokeTestFunction(BTestRunner.java:601)
	at org.ballerinalang.test.runtime.BTestRunner.executeFunction(BTestRunner.java:436)
	at org.ballerinalang.test.runtime.BTestRunner.lambda$executeTests$2(BTestRunner.java:332)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.ballerinalang.test.runtime.BTestRunner.executeTests(BTestRunner.java:320)
	at org.ballerinalang.test.runtime.BTestRunner.execute(BTestRunner.java:291)
	at org.ballerinalang.test.runtime.BTestRunner.runTest(BTestRunner.java:106)
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
Caused by: java.lang.NullPointerException
	at roland.asb.0_1_0.tests.main_test.testReceiveFromQueueOperation(tests/main_test.bal:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.lambda$runOnSchedule$0(TesterinaFunction.java:109)
	at org.ballerinalang.jvm.scheduling.SchedulerItem.execute(Scheduler.java:500)
	at org.ballerinalang.jvm.scheduling.Scheduler.run(Scheduler.java:276)
	at org.ballerinalang.jvm.scheduling.Scheduler.runSafely(Scheduler.java:244)
	at java.lang.Thread.run(Thread.java:748)
 
[2020-11-27 10:46:51,105] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 10:55:03,887] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 10:57:17,868] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 10:59:06,457] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 11:00:31,753] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 11:05:31,009] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 11:32:59,572] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 11:36:25,938] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 11:46:02,003] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 13:38:49,501] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 15:12:34,275] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 15:21:10,557] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 15:33:38,429] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 15:43:41,025] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 15:57:10,849] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 15:58:39,666] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 16:00:28,725] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 16:33:14,804] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 16:36:42,887] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 16:39:32,492] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 16:43:21,374] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 17:02:57,200] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-27 17:43:41,745] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-30 08:59:18,697] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
[2020-11-30 09:06:07,706] SEVERE {b7a.log.crash} - there are test failures 
java.lang.RuntimeException: there are test failures
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:77)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
 
